tapir/shifts/tests/test_shiftwatch_forms.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from django.urls import reverse
+
+from tapir.shifts.models import StaffingEventsChoices, ShiftWatch
+from tapir.shifts.tests.factories import ShiftFactory
+from tapir.utils.tests_utils import TapirFactoryTestBase
+
+
+class TestShiftCancel(TapirFactoryTestBase):
+
+    @pytest.mark.django_db
```

Comment:
I think we don't need this since we already inherit from `django.test.testcases.TestCase`